docs/reference/supervisor.md

```diff
@@ -1164,13 +1164,17 @@ The `sampling_period` argument determines how often the contact point data shoul
 The `wb_supervisor_node_disable_contact_point_tracking` function disables contact point data tracking.
 
 The `wb_supervisor_node_get_contact_point` function returns the contact point with given index in the contact point list of the given `Solid`.
+**Deprecation warning:** This function is deprecated and the `wb_supervisor_node_get_contact_points` function should be used instead.
```

Comment:
Deprecated functions should be removed from the reference manual.

Reply:
But the differential drive wasn't deleted (https://www.cyberbotics.com/doc/reference/differentialwheels), neither was `robot.get[DEVICE_TYPE]` (https://cyberbotics.com/doc/reference/robot?tab-language=python#wb_robot_get_device). I am not sure what is the best way to deal with deprecated functions.

Reply:
I believe removing them from the documentation is one of the best way to discourage their use. So, I would go ahead a remove them from the documentation. Differential drive and `robot.get[DEVICE_TYPE]` should be deleted in a separate PR.

Reply:
We always deleted all the deprecation functions from the documentation.
In these two cases it is possible that we forgot to do it.

Reply:
DifferentialWheels is already deleted from the develop branch. I will take care of deleting the python `robot.get[DEVICE_TYPE]` now.